pkg/rbac/permission_map.go

```diff
@@ -28,6 +28,7 @@ type Resource string
 
 const (
 	ResourceRepositories Resource = "repositories"
+	ResourceTemplates             = "templates"
```

Comment:
does permission_map_test.go need to be updated as well?

Reply:
yes it does!